languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,17 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rule id="IL_FAUT" name="Il faut">
+          <antipattern>
+              <token postag="M fin inte"></token>
+          </antipattern>
+          <pattern>
+              <token postag="SENT_START"/>
+              <marker>     <token>Faut</token></marker>
+          </pattern>
+          <message>C'est une expression familière incorrecte au niveau grammatical. Souhaitez-vous écrire <suggestion>Il <match no="2" case_conversion="startlower"/></suggestion> ?</message>
```

Comment:
This message is not quite the way I like it. If a colloquial expression doesn't correspond to the grammar rules of the standard language, it doesn't mean it is categorically "agrammatical".

I'd prefer something like "Cette expression relève d'un registre familier. Dans un contexte formel, il convient d'écrire "Il faut".